Blueprints/BlueprintDefinitions/vs2022/EmptyServerless/template/src/BlueprintBaseName.1/Functions.cs

```diff
@@ -20,19 +22,24 @@ public Functions()
     /// <summary>
     /// A Lambda function to respond to HTTP Get methods from API Gateway
     /// </summary>
-    /// <param name="request"></param>
-    /// <returns>The API Gateway response.</returns>
-    public APIGatewayProxyResponse Get(APIGatewayProxyRequest request, ILambdaContext context)
+    /// <remarks>
+    /// This uses the <see href="https://github.com/aws/aws-lambda-dotnet/blob/master/Libraries/src/Amazon.Lambda.Annotations/README.md">Lambda Annotations</see> 
+    /// programming model to bridge the gap between the Lambda programming model and a more idiomatic .NET model.
+    /// 
+    /// This automatically handles reading parameters from an APIGatewayProxyRequest
+    /// as well as syncing the function definitions to serverless.template each time you build.
+    /// 
+    /// If you do not wish to use this model and need to manipulate the API Gateway 
+    /// objects directly, see the accompanying Readme.md for instructions.
+    /// </remarks>
+    /// <param name="context">Information about the invocation, function, and execution environment</param>
+    /// <returns>The response as an implicit <see cref="APIGatewayProxyResponse"/></returns>
+    [LambdaFunction(PackageType = LambdaPackageType.Zip, Policies = "AWSLambdaBasicExecutionRole", MemorySize = 256, Timeout = 30)]
```

Comment:
Since `Zip` is the default I think we can make this look cleaner and not have the `PackageType = LambdaPackageType.Zip`. Just have it for the image based template.

Reply:
Done